src/uploader.ts

```diff
@@ -62,7 +62,11 @@ export class Uploader {
 
   private executeUpload() {
     const lxrConfig = require(this.pathHelper.getLxrConfigPath());
-    console.log(chalk.yellow(chalk.italic(`Uploading to ${lxrConfig.host} ...`)));
+    if (lxrConfig && lxrConfig.hasOwnProperty('proxyURL')) {
+      // Set the PROXY_URL envvar
+      process.env.PROXY_URL = lxrConfig.proxyURL
```

Comment:
Why do we set the envvar? Is it required for something in nodejs to work with the proxy?